cmd/project.go

```diff
@@ -8,8 +8,8 @@ import (
 
 var projectCmd = &cobra.Command{
 	Use:   "project [command]",
-	Short: "manage projects",
-	Long:  "Project management for Runpod projects",
+	Short: "Manage RunPod projects",
+	Long:  "Rapidly develop and deploy a project entirely on RunPod's infrastructure without touching Docker",
```

Comment:
I like most of these edits, but I feel like this one is kinda sales pitchy

Reply:
How about "Develop and deploy a project entirely on RunPod's infrastructure"?

Reply:
That's nice, maybe "Develop and deploy projects" since we used projects plural in the short version